Sources/MongoSwift/SDAM.swift

```diff
@@ -3,12 +3,21 @@ import Foundation
 
 /// A struct representing a network address, consisting of a host and port.
 public struct ServerAddress: Equatable {
+    public enum HostType: String {
+        case ipv4
+        case ip_literal
+        case hostname
+        case unix
```

Comment:
Can this be expanded to `unixDomainSocket` to be more specific?

Reply:
done